test/unit/vdom/skip.js

```diff
@@ -10,6 +10,7 @@ describe('vdom/skip', () => {
       },
       render() {
         vdom.element('div', { skip: true }, () => {
+          vdom.text('text');
```

Comment:
`render() {
        vdom.element('div', { skip: true }, () => {
          vdom.element('span', () => {
            vdom.text('text');
          });
          vdom.element('span', () => {
            vdom.text('text');
          });
        });
      },`

render function like this will fail the test